### PR TITLE
[explorer]: update APPS panel (active connections)

### PR DESCRIPTION
**Pull request by plam-ml**
## Description 

- https://mysten.atlassian.net/browse/APPS-693

Screenshot 2023-04-17 at 1 41 44 PM

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/ui/app/shared/text/index.tsx [hunk 1]
```diff
@@ -58,6 +58,9 @@ const textStyles = cva([], {
             true: 'font-mono',
             false: 'font-sans',
         },
+        lineHeight: {
```
Comment (plam-ml) [inline apps/wallet/src/ui/app/shared/text/index.tsx:61]: looking for suggestion here. The lineheight is really crammed when text is wrapped

Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/shared/text/index.tsx:61]: You shouldn't need to do this, I think `p2` is the correct text config (we need to update our naming to align with the pbodysmall in figma)

Comment (plam-ml) [reply apps/wallet/src/ui/app/shared/text/index.tsx]: Ah ok thanks. I was trying to fit bodySmall so it was not working correctly

Comment (Jordan-Mysten) [reply apps/wallet/src/ui/app/shared/text/index.tsx]: Opened a PR to rename these classes to re-align with the figma: https://github.com/MystenLabs/sui/pull/11010

**Review by Jordan-Mysten — APPROVED** [review]

**Comment by plam-ml** [comment]
pending on https://github.com/MystenLabs/sui/pull/11010 to be merged

**Pull request merged**